<?xml version="1.0"?>
<launch>
	<arg name="joydev" default="/dev/input/js2" />
	
  <!-- Launch joy_node that publishes Joy messages -->
  <node name="joy" type="joy_node" pkg="joy" >
    <param name="dev" value="$(arg joydev)"/>
  </node>

  <!-- Launch joyop_cmdvel to teleoperate robots-->
  <node name="cmd_vel_joy" type="joyop_cmdvel" pkg="robo_mower" output='screen' >
		<param name="propel_scale" value="0.3"/>
		<param name="steer_scale" value="0.8"/>
		<param name="output_frequency" value="15"/>
		<param name="joy_topic" value="/joy"/>
	</node>

</launch>



